src/main/java/seedu/cafectrl/command/ExitCommand.java

```diff
@@ -0,0 +1,13 @@
+package seedu.cafectrl.command;
+
+import seedu.cafectrl.data.Menu;
+import seedu.cafectrl.ui.Ui;
+
+public class ExitCommand extends Command{
```

Comment:
Noticed missing spacing before curly bracket {

Reply:
oops thank you for catching that